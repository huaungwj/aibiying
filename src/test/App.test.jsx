import App from "../router/index";
import { render } from "@testing-library/react"

describe("test", () => {
    it("app", () => {
        let app = render(<App />);
        // console.log(app);
    });
});
