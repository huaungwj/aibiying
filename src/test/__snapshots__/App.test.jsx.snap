// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="navBarContainer"
  >
    <div
      class="navBarLogo"
    >
      <a
        href="/"
      >
        <img
          src="logo.jpg"
        />
      </a>
    </div>
    <div
      class="navBarSearchBox"
    >
      <button
        class="navBarSearchButton"
      >
        <input
          class="navBarSearchInput"
          placeholder="start your search"
          value=""
        />
        <div
          class="navBarSearchSubmit"
        >
          <span
            aria-label="search"
            class="anticon anticon-search"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="search"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
              />
            </svg>
          </span>
        </div>
      </button>
    </div>
    <div
      class="navBarRightNav"
    >
      <ul>
        <li
          class="liActive"
        >
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class=""
        >
          <a
            data-testid="House"
            href="/house"
          >
            House
          </a>
        </li>
        <li
          class=""
        >
          <a
            href="/booking"
          >
            Booking
          </a>
        </li>
      </ul>
      <div
        class="personalCenter"
      >
        <button
          data-testid="user-avater"
        >
          <span
            aria-label="menu"
            class="anticon anticon-menu"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="menu"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"
              />
            </svg>
          </span>
          <svg
            aria-hidden="true"
            class="personalSvg"
            focusable="false"
            role="presentation"
            style="display: block; height: 100%; width: 100%; fill: currentcolor;"
            viewBox="0 0 32 32"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="m16 .7c-8.437 0-15.3 6.863-15.3 15.3s6.863 15.3 15.3 15.3 15.3-6.863 15.3-15.3-6.863-15.3-15.3-15.3zm0 28c-4.021 0-7.605-1.884-9.933-4.81a12.425 12.425 0 0 1 6.451-4.4 6.507 6.507 0 0 1 -3.018-5.49c0-3.584 2.916-6.5 6.5-6.5s6.5 2.916 6.5 6.5a6.513 6.513 0 0 1 -3.019 5.491 12.42 12.42 0 0 1 6.452 4.4c-2.328 2.925-5.912 4.809-9.933 4.809z"
            />
          </svg>
        </button>
        <div
          class="personalSelect"
          style="display: none;"
        >
          <ul>
            <li
              class="PersonalLiFirst"
            >
              Login
            </li>
            <li>
              logout
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    class="relative banner bg-no-repeat bg-fixed bg-cover"
  >
    <div
      class="max-w-screen-lg mx-auto"
    >
      <div
        class="flex flex-row flex-grow w-full justify-center space-x-4 items-center bg-gray-100 rounded-full relative top-10 hidden lg:inline-flex xl:inline-flex m-5"
      >
        <div
          class=" px-5 py-3  rounded-full hover:bg-gray-200 hover:shadow-lg cursor-pointer"
        >
          <h2
            class="text-sm font-bold text-gray-700"
          >
            Location
          </h2>
          <span
            class="text-gray-600 text-sm"
          >
            Where are you going ?
          </span>
        </div>
        <div
          class=" px-8 py-3  rounded-full hover:bg-gray-200 hover:shadow-lg cursor-pointer"
        >
          <h2
            class="text-sm font-bold text-gray-700"
          >
            Check in
          </h2>
          <span
            class="text-gray-600 text-sm"
          >
            Add Dates
          </span>
        </div>
        <div
          class=" px-8 py-3  rounded-full hover:bg-gray-200 hover:shadow-lg cursor-pointer"
        >
          <h2
            class="text-sm font-bold text-gray-700"
          >
            Check out
          </h2>
          <span
            class="text-gray-600 text-sm"
          >
            Add Dates
          </span>
        </div>
        <div
          class=" px-8 py-3  rounded-full hover:bg-gray-200 hover:shadow-lg cursor-pointer"
        >
          <h2
            class="text-sm font-bold text-gray-700"
          >
            other
          </h2>
          <span
            class="text-gray-600 text-sm"
          >
            screen
          </span>
        </div>
        <a
          href="/search"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root searchIconDate bg-red-500 text-white rounded-full p-1"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
        </a>
      </div>
      <div
        class="bannerInfo px-5 py-2"
      >
        <h5
          class="text-white font-bold text-xl pb-3"
        >
          Airbnb 
          <span
            class="bg-gray-900 text-white rounded-lg p-1"
          >
            2021
          </span>
        </h5>
        <h1
          class="text-white font-semibold text-4xl lg:text-5xl xl:text-5xl"
        >
          Introducing 100+ 
          <br />
           upgrades across 
          <br />
          our entire service
        </h1>
        <a
          href="/search"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text w-40 bannerButton my-4"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Learn whats new
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </a>
      </div>
    </div>
  </div>
  <main
    class="max-w-screen-lg mx-auto"
  >
    <h1
      class="text-gray-900 font-bold text-3xl py-4 m-5"
    >
      Live anywhere
    </h1>
    <div
      class="grid grid-flow-row-dense md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-3 place-items-center gap-16"
    >
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://a0.muscache.com/im/pictures/2f13349d-879d-43c6-83e3-8e5679291d53.jpg?im_w=320"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          Outdoor getaways
        </h1>
      </div>
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://a0.muscache.com/im/pictures/36f53e61-db8d-403c-9122-5b761c0e4264.jpg?im_w=320"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          Unique stays
        </h1>
      </div>
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://a0.muscache.com/im/pictures/7d82ca14-56e5-4465-8218-dcfa7d69b6ac.jpg?im_w=320"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          Entire homes
        </h1>
      </div>
    </div>
    <div
      class="grid grid-flow-row-dense grid-cols-1 relative w-full h-92"
    >
      <img
        alt="bannerImage"
        class="w-full h-92 my-10 rounded-xl shadow-lg object-cover"
        src="https://a0.muscache.com/im/pictures/38100f6a-9a35-45b8-a503-cfe384d953b5.jpg?im_w=960"
      />
      <div
        class="absolute top-12 md:top-40 lg:top-40 xl:top-40 mx-10 my-15 p-5"
      >
        <h1
          class="text-3xl md:text-6xl lg:text-6xl xl:text-6xl text-gray-900 font-semibold my-2 md:my-4 lg:my-4 xl:my-4 "
        >
          The Greatest 
          <br />
           Outdoors
        </h1>
        <h3
          class="hidden md:block lg:block xl:block  text-lg text-gray-800"
        >
          Wishlists curated by Airbnb
        </h3>
        <button
          class="tranistion duration-700 ease-in-out bg-gray-900 text-white px-5 py-3 rounded-lg my-4 hover:bg-white hover:text-gray-900 hover:font-semibold"
        >
          Get inspired
        </button>
      </div>
    </div>
    <div
      class="m-5"
    >
      <h1
        class="text-3xl text-gray-900 font-bold md:text-4xl lg:text-4xl xl:text-4xl py-3 "
      >
        Discover Experiences
      </h1>
      <h4
        class="font-light text-2xl text-gray-500 pb-4"
      >
        Unique activities with local experts—in person or online.
      </h4>
    </div>
    <div
      class="grid grid-flow-row-dense md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-3 place-items-center gap-16"
    >
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://i.ibb.co/NNrMPcq/1.jpg"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          Featured collection:Wanderlust
        </h1>
      </div>
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://i.ibb.co/qBY6b7t/2.jpg"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          0nline Experiences
        </h1>
      </div>
      <div>
        <div
          class="rounded-lg overflow-hidden w-80 mx-2 shadow-xl cursor-pointer"
        >
          <img
            alt="images"
            class="transition duration-700 ease-in-out rounded-lg transform hover:scale-125 object-cover"
            src="https://i.ibb.co/vqfJ8MT/3.jpg"
          />
        </div>
        <h1
          class="text-gray-900 font-semibold text-xl text-center my-3"
        >
          Experiences
        </h1>
      </div>
    </div>
    <div
      class="grid grid-flow-row-dense grid-cols-1 relative w-full h-92"
    >
      <img
        alt="bannerImage"
        class="w-full h-92 my-10 rounded-xl shadow-lg object-cover"
        src="https://i.ibb.co/YRmwgBt/b.jpg"
      />
      <div
        class="absolute top-0 md:top-30 lg:top-30 xl:top-30 mx-10 my-10 md:my-20 lg:my-20 xl:my-20p-5"
      >
        <h1
          class="text-3xl md:text-6xl lg:text-6xl xl:text-6xl text-white font-semibold my-2 md:my-4 lg:my-4 xl:my-4 "
        >
          Become a Host
        </h1>
        <h3
          class="hidden md:block lg:block xl:block  text-lg text-white"
        >
          Earn extra income and get opportunities
        </h3>
        <button
          class="tranistion duration-700 ease-in-out bg-white text-gray-900 px-5 py-3 rounded-lg my-4 hover:bg-gray-900 hover:text-white white  font-bold hover:font-bold cursor-pointer"
        >
          Learn more
        </button>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
